examples/arduino/sketches/sketch-lidar-printDistance.ino

```diff
@@ -0,0 +1,23 @@
+#include <Smartcar.h>
+#include <Wire.h>
+#include <VL53L0X.h>
+
+VL53L0X sensor;
+void setup()
+{
+  Serial.begin(9600);
+  Wire.begin();
+
+  sensor.setTimeout(500);
+  if (!sensor.init())
+  {
+    Serial.println("Failed to detect and initialize sensor!");
+    while (1) {}
```

Comment:
this would lock up bad (do `while(!sensor.init())`)

Reply:
Wdym "lock up"? It simply invokes UB, so you have no idea what it actually does